app/src/main/java/com/concordium/wallet/data/preferences/AuthPreferences.kt

```diff
@@ -142,17 +146,57 @@ class AuthPreferences(val context: Context) :
         setString(SEED_PHRASE, "")
     }
 
-    fun setSeedPhrase(value: String) {
+    suspend fun setSeedPhrase(value: String, password: String): Boolean {
```

Comment:
I suggest modifying the name of the method, so it reflects the intent.
From what I can tell, it takes a value, encodes that to a HEX format representing the seed.
Then it encrypts the seed, before returning the encrypted version. 

So consider renaming the method to GetEncryptedSeed

However the SetStringWithResult confuses me, why does it take 2 parameters, and what does it do? :)

Reply:
SetStringWithResult saves a string to shared preference and returns result if the write was successful or not, the first parameter is the key and the second the value.

The setSeedPhrase actually returns Boolean if the seed is successfully encrypted and then successfully stored. 
First we try to encrypt the seed, if encrypted seed is null return false.
if the encrypted seed is not null, try to save it. If it is saved successful return true, if not false.

Reply:
Alright, I guess I understand now. So name should be tryToSetEncryptedSeedPhrase to express intent.

Reply:
Ok, will change that